src/main/java/com/iexec/sms/secret/app/OwnerRole.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.sms.secret.app;
+
+/**
+ * Defines which role a tee task runtime secret owner can have.
+ */
+public enum OwnerRole {
```

Comment:
Do you find it useful to add `Secret` there? I think that `OwnerRole` is enough to describe what's expected, without having to say what's owned.

Reply:
I was also wondering a lot about that. I like SecretOwnerRole since it reminds it has nothing to do with what is deployed or anything else, it's just about secrets here.
The only food reason of using OwnerRole is reusing that enum in a another context.
Current context is so sensitive that it worths declaring "explicit" names in order to get descriptions as precise as it should.

Reply:
If consensus goes for a more describing name and we're sure we don't want to use it in another context, let's rename it!